tests/test_converter.py

```diff
@@ -0,0 +1,10 @@
+from mindmeld.test import TestConversation
```

Comment:
move this file to https://github.com/cisco/mindmeld/blob/master/tests/converter/d

Reply:
done